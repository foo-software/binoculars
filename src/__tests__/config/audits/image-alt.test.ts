import auditImageAlt from '../../../config/audits/image-alt';

describe('image-alt audit', () => {
  it('should match snapshot', () => {
    expect(auditImageAlt('en')).toMatchSnapshot();
  });
});
