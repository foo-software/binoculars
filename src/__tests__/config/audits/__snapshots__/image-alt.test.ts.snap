// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`image-alt audit should match snapshot 1`] = `
Object {
  "description": "Informative elements should aim for short, descriptive alternate text. Decorative elements can be ignored with an empty alt attribute. [Learn more](https://web.dev/image-alt/).",
  "title": "Image elements have \`[alt]\` attributes",
}
`;
